test/mockups/element-mock.js

```diff
@@ -160,12 +160,18 @@ function setAttribute(attr, value) {
         attrObj = {
             name: attr,
             value: value,
-            ownerElement: this
+            ownerElement: this,
+            get nodeValue() {
```

Comment:
I can't review these. Don't have knowledge. just make note to review with chuck

Reply:
Ok will check it with him